src/Sulu/Bundle/DocumentManagerBundle/DependencyInjection/Configuration.php

```diff
@@ -43,14 +43,27 @@ public function getConfigTreeBuilder()
                 ->arrayNode('mapping')
                     ->useAttributeAsKey('alias')
                     ->prototype('array')
-                    ->children()
-                        ->scalarNode('class')
-                            ->info('Fully qualified class name for mapped object')
-                            ->isRequired()
-                        ->end()
-                        ->scalarNode('phpcr_type')
-                            ->info('PHPCR type to map to')
-                            ->isRequired()
+                        ->children()
+                            ->scalarNode('class')
+                                ->info('Fully qualified class name for mapped object')
+                                ->isRequired()
+                            ->end()
+                            ->scalarNode('phpcr_type')
+                                ->info('PHPCR type to map to')
+                                ->isRequired()
+                            ->end()
+                            ->arrayNode('mapping')
```

Comment:
Is that a new feature? I can't find any corresponding changes in the document manager...